bin/accountsandtransactions

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
```

Comment:
I would recommend sticking this file in `examples/account_and_transactions.rb`. When you write this as a bin stub, you can actually run from their production server, which we probably don't want. Creating an examples directory helps solve that problem. At least, that's what I've found to be most helpful.

Reply:
I think you should make the same changes to the rest of the binstubs in this project.